### PR TITLE
removed deprecated Twig features

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | yes |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Resources/config/schema/twig-1.0.xsd [hunk 1]
```diff
@@ -9,8 +9,6 @@
 
     <xsd:complexType name="config">
         <xsd:sequence>
-            <!-- @deprecated since version 2.6, to be removed in 3.0 -->
-            <xsd:element name="form" type="form" minOccurs="0" maxOccurs="1" />
```
Comment (Tobion) [inline src/Symfony/Bundle/TwigBundle/Resources/config/schema/twig-1.0.xsd]: It looks like you can also remove the ´complexType name="form"`

Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/Resources/config/schema/twig-1.0.xsd]: good catch, fixed

File: src/Symfony/Bridge/Twig/AppVariable.php [hunk 2]
```diff
@@ -25,20 +24,11 @@
  */
 class AppVariable
 {
-    private $security;
```
Comment (Tobion) [inline src/Symfony/Bridge/Twig/AppVariable.php]: this property is still referenced in `getUser`

**Comment by Tobion** [comment]
apart from comment above, :+1:

File: src/Symfony/Bridge/Twig/AppVariable.php (hunk 2 above)
Comment (fabpot) [inline src/Symfony/Bridge/Twig/AppVariable.php]: fixed

**Pull request merged**